src/QMCWaveFunctions/tests/test_RotatedSPOs_LCAO.cpp

```diff
@@ -90,7 +90,8 @@ std::string setupRotationXML(const std::string& rot_angle_up,
           </basisGroup>
         </atomicBasisSet>
       </basisset>
-     <sposet basisset="LCAOBSet" name="spo-up" size="2" optimize="yes">)";
+     <rotated_sposet name="rot-spo-up">
+       <sposet basisset="LCAOBSet" name="spo-up">)";
```

Comment:
Note that there are two implicit assumptions
1. the LCAOSet size is derived from the basisset.
2. the rotated_sposet size is derived from the original sposet size.

Reply:
Provided the revised input is able to handle e.g. reruns of Josh's FeO and for Anouar to be able to OO small molecules, we are fine. 

Any remaining concerns?

Reply:
LGTM.